### ISSUE TITLE
SendEmailVerificationAsync doesn`t work after update from 60 to 71 version

### ISSUE THREAD

**Issue by oleksiyyurchyna** [xamarin/GooglePlayServicesComponents#263]
### Xamarin.Android Version (eg: 6.0):
71.1605

### Operating System & Version (eg: Mac OSX 10.11):
Windows

### Describe your Issue:
Once updated from version 60 to 71, my mail verification sending stopped working:

        public async Task SendEmailVerificationAsync()
        {
            await FirebaseAuth.Instance.CurrentUser.SendEmailVerificationAsync(ActionCodeSettings.NewBuilder().Build());
        }

After back to 60 - it is fine again.

**Comment by moljac** [comment]
@oleksiyyurchyna Could you provide a minimal repro sample, please?

Just link it here. It will reduce time to reproduce the issue.

**Comment by oleksiyyurchyna** [comment]
Hi,

the error I receive is:
Java.Lang.IllegalArgumentException: 'Cannot build ActionCodeSettings with null URL. Call #setUrl(String) before calling build()'

Below is the code I have which failing for 71 and not failing for 60:

var firebase = DependencyService.Get<IFirebaseAuthenticator>();

            try
            {
                var token = await firebase.SignInWithEmailAndPasswordAsync(EmailEntry.Text, PasswordEntry.Text);

                var isVerified = await firebase.IsCurrentUserEmailVerified();
                if (!isVerified)
                {
                    await firebase.SendEmailVerificationAsync();
                    return;
                }
            }
            catch (Exception ex)
            {
                await DisplayAlert("error", ex.Message, "OK");
            }

Implementations on Android side:

        public async Task<string> SignInWithEmailAndPasswordAsync(string email, string password)
        {
            var authResult = await FirebaseAuth.Instance.SignInWithEmailAndPasswordAsync(email, password);

            var getTokenResult = await authResult.User.GetIdTokenAsync(false);
            return getTokenResult.Token;
        }


        public async Task<bool> IsCurrentUserEmailVerified()
        {
           await FirebaseAuth.Instance.CurrentUser.ReloadAsync();

            return FirebaseAuth.Instance.CurrentUser.IsEmailVerified;
        }


        public async Task SendEmailVerificationAsync()
        {
            await FirebaseAuth.Instance.CurrentUser.SendEmailVerificationAsync(ActionCodeSettings.NewBuilder().Build());
        }

**Comment by moljac** [comment]
@oleksiyyurchyna 

Well in your code I can see call to `Build()`, but no calls to `SetUrl()` (OK that might be a property `Url`).

Read what error says...

Try calling managed equivalent of `setUrl`... API might have changed (well it changed for sure)

**Comment by oleksiyyurchyna** [comment]
@moljac 
Yes, it is clear for me that I set no URL and then receive "no URL is set" error. 
I have created the ticket in a chance to get know how can I send the verification mail in version 71.
Which URL I can put there? Because in version 60 there was no need to put URL.

**Comment by moljac** [comment]
@oleksiyyurchyna 
Could you create minimal repro sample please? That would help a lot.

**Comment by oleksiyyurchyna** [comment]
@moljac 
Below you can find the repository for test
https://github.com/oleksiyyurchyna/reprosample
